tokens/dist/css/spectrum/custom-large-vars.css

```diff
@@ -101,7 +101,8 @@
   --spectrum-assetcard-header-content-font-size:var(--spectrum-heading-size-xxs);
   --spectrum-assetcard-content-font-size:var(--spectrum-body-size-xs);
 
-  --spectrum-tooltip-animation-distance:5px;
+  /* To get a square that measures 12px on the diagonal, the sides have to be 10px */
+	--spectrum-tooltip-tip-square-size:10px;
```

Comment:
To clarify, we're back to the same values as we had at the beginning for this, design and dev both agreed that these were the most accurate 🙃